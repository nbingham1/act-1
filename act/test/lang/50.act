defproc foo(bool x, y)
{
  bool w;
  prs {
   x <5,3,lvt> -> x-
   w -> x-

   passn<5,lvt,lvt>(x,x,y)
  }
}

foo f;
