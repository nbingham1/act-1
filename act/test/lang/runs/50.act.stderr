ERROR: File `50.act', line 8, col 10
	Multiple device flavors?
